singularity-eos/eos/eos_vinet.hpp

```diff
@@ -304,6 +306,14 @@ PORTABLE_INLINE_FUNCTION Real Vinet::PressureFromDensityInternalEnergy(
   Vinet_F_DT_func(rho, temp, output);
   return output[1];
 }
+PORTABLE_INLINE_FUNCTION Real Vinet::MinInternalEnergyFromDensity(const Real rho,
+                                                                  Real *lambda) const {
+#ifndef NDEBUG
```

Comment:
could probably replace with the function @jhp-lanl

Reply:
done